### PR TITLE
Add '/api' path all the apis in backend

### PR DESCRIPTION
**Pull request by rantan**
This is for deployment.

To recognize to be requests for backend in a load balancer. 
Some load balancers like ALB  are able to recoginize path to switch to forward the requests to targets.

### CODE REVIEW

**Review by azuchi — CHANGES_REQUESTED** [review]
I think `api` is better than `backend`, in fact it is api.

**Comment by rantan** [comment]
Ok, I will change to 'api'

File: frontend/src/app/backend.service.ts [hunk 1]
```diff
@@ -14,55 +14,55 @@ export class BackendService {
...
   }
 
   getBlock(blockHash: string): Observable<any> {
-    return this.http.get(`${this.backendUrl}/block/${blockHash}`);
+    return this.http.get(`${this.backendUrl}/api/block/${blockHash}`);
   }
 
   getRawBlock(blockHash: string): Observable<any> {
-    return this.http.get(`${this.backendUrl}/block/${blockHash}/raw`);
+    return this.http.get(`${this.backendUrl}/backend/block/${blockHash}/raw`);
```
Comment (azuchi) [inline frontend/src/app/backend.service.ts:33]: `backend` still remain.

Comment (rantan) [reply frontend/src/app/backend.service.ts]: done

**Review by azuchi — APPROVED** [review]

**Pull request merged**